integration/loki_micro_services_delete_test.go

```diff
@@ -193,6 +193,7 @@ func TestMicroServicesDeleteRequest(t *testing.T) {
 
 	// Query lines
 	t.Run("query again to verify logs being served from storage", func(t *testing.T) {
+		time.Sleep(time.Millisecond * 3500)
```

Comment:
I don't think this is the correct approach to get these tests to pass.

We could use `require.Eventually`, but that is outside the scope of this PR. Please remove this.

Reply:
ok

Reply:
> We could use `require.Eventually`

OK，let me try how to use this